infrastructure/docs/docker/README.md

```diff
@@ -1,7 +1,7 @@
 ## Docker guide
 
 First you need to install Docker (https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#install-docker-ce-1)
-When you have installed Docker you need to create a Dockerfile, its content will depend on the technology, for example, in .NET Core we use [this definition](https://github.com/Wolox/netcore-bootstrap/blob/master/NetCoreBootstrap/Dockerfile) and in RoR we use [this definition](https://github.com/Wolox/rails-bootstrap/blob/master/Dockerfile)
+When you have installed Docker you need to create a Dockerfile, its content will depend on the technology, for example, in .NET Core we use [this definition](https://github.com/Wolox/netcore-bootstrap/blob/master/src/Dockerfile) and in RoR we use [this definition](https://github.com/Wolox/rails-bootstrap/blob/master/Dockerfile)
```

Comment:
I'd say
"Once you have Docker installed, you'll need to create a Dockerfile. Its content will depend on the technology, for example,.... "

Reply:
cool :D